cmd/notary/main.go

```diff
@@ -45,17 +51,27 @@ func parseConfig() {
 		if homeDir == "" {
 			fatalf("cannot get current user home directory")
 		}
-		trustDir = filepath.Join(homeDir, filepath.Dir(defaultTrustDir))
+		trustDir = filepath.Join(homeDir, filepath.Dir(configDir))
 
 		logrus.Debugf("no trust directory provided, using default: %s", trustDir)
 	} else {
 		logrus.Debugf("trust directory provided: %s", trustDir)
 	}
 
-	// Setup the configuration details
+	// If there was a commandline configFile set, we parse that.
+	// If there wasn't we attempt to find it on the default location ~/.notary/config
+	if configFile != "" {
+		configFileExt = strings.TrimPrefix(filepath.Ext(configFile), ".")
+		configFileName = strings.TrimSuffix(filepath.Base(configFile), filepath.Ext(configFile))
```

Comment:
Would this mean the config file name would end with a `.`?  Since the `.` is trimmed off the `configFileExt`?